docs/extras/code_samples/workflow_execution.txt

```diff
@@ -0,0 +1,21 @@
+from mindee import Client, WorkflowResponse
+
+# Init a new client
+mindee_client = Client(api_key: "my-api-key")
+
+workflow_id = "workflow-id"
+
+# Load a file from disk
+input_doc = mindee_client.source_from_path("/path/to/the/file.ext")
+
+# Send the file to the workflow.
+result: WorkflowResponse = mindee_client.execute_workflow(
+    input_doc,
+    workflow_id,
+    # Optionally, add an alias and a priority to the workflow.
+    # alias="my-alias",
+    # priority=ExecutionPriority.LOW
```

Comment:
import the `ExecutionPriority` class?

Reply:
Well, it's commented out so I reasoned it wouldn't be necessary, but sure :+1:

Reply:
was more of a "do you think we should?", but yes, better I think ;-)